pandas/tests/io/test_excel.py

```diff
@@ -691,25 +691,19 @@ def test_read_from_http_url(self, ext):
         local_table = self.get_exceldf('test1', ext)
         tm.assert_frame_equal(url_table, local_table)
 
-    @td.skip_if_no("s3fs")
     @td.skip_if_not_us_locale
-    def test_read_from_s3_url(self, ext):
-        moto = pytest.importorskip("moto")
-        boto3 = pytest.importorskip("boto3")
-
-        with moto.mock_s3():
-            conn = boto3.resource("s3", region_name="us-east-1")
-            conn.create_bucket(Bucket="pandas-test")
-            file_name = os.path.join(self.dirpath, 'test1' + ext)
-
-            with open(file_name, "rb") as f:
-                conn.Bucket("pandas-test").put_object(Key="test1" + ext,
-                                                      Body=f)
-
-            url = ('s3://pandas-test/test1' + ext)
-            url_table = read_excel(url)
-            local_table = self.get_exceldf('test1', ext)
-            tm.assert_frame_equal(url_table, local_table)
+    def test_read_from_s3_url(self, ext, s3_resource):
```

Comment:
same why are you not mocking this?

Reply:
@jreback, same as above, the mocking is done in the `s3_resource` fixture that I added to the sig